.github/workflows/ci.yml

```diff
@@ -38,7 +38,14 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: yarn
       - run: yarn --immutable --network-timeout 1500000
-      - run: yarn lerna run test:integration
+      - name: Run cli package integration tests
+        run: |
+          cd packages/cli
+          yarn test:integration
+      - name: Run orgs-v5 integration tests
+        run: |
+          cd packages/orgs-v5
+          yarn test:integration
```

Comment:
have you considered using the `[--cwd](https://classic.yarnpkg.com/lang/en/docs/cli/#toc-cwd) flag here?

Reply:
No. I didn't do much optimization here, honestly. We're going to get rid of the orgs-v5 tests in the near future and all I really wanted was to show the output from the cli integration tests. But thank you!